Vagrantfile

```diff
@@ -0,0 +1,21 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure(2) do |config|
+  config.vm.box = "ubuntu/trusty64"
+  config.vm.network "private_network", ip: "192.168.33.10"
```

Comment:
I'm not forcing any IP address by default, just exposing the default HASS port, it works alright for me:
```
Vagrant.configure(2) do |config|
  config.vm.box = "debian/contrib-jessie64"
  config.vm.network "forwarded_port", guest: 8123, host: 8123
  config.vm.provision "shell" do |shell|
    shell.path = "provision.sh"
  end
end
```